### ISSUE TITLE
sound source file only ogg format ?  can wav?

### ISSUE THREAD

**Issue by marshal-it** [eXifreXi/eXiSoundVis#15]

**Comment by xSiraxXx** [comment]
You'd need to program the .wav part yourself since SoundVis only works with .ogg files.

**Comment by eXifreXi** [comment]
Yeah the wav support can be added if we analyze the files first and save it on import.
I saw a plugin from someone else doing that, but I have no time implementing it.
It's overriding the Importer for WAV files and makes it create a new uasset that holds the array data.
No idea how performant it is to save such a big array though.
